test/xpu_api/utilities/meta/meta.trans/meta.trans.other/result_of.pass.cpp

```diff
@@ -0,0 +1,186 @@
+// -*- C++ -*-
```

Comment:
Should this test be renamed to `invoke_result.pass.cpp` since we test this trait?
`result_of` is a different trait in STL that is not used in this test.

Reply:
Fixed